### PR TITLE
Add PHP8 compatibility

### LINKED ISSUE #137: Fatal error in php8
```
Fatal error: Uncaught Error: filter_input(): Argument #4 ($options) must be of type array|int, string given
in /home/pross/public_html/wp-content/plugins/classicpress-seo/includes/helpers/class-param.php on line 28

Call stack:

filter_input()
wp-content/plugins/classicpress-seo/includes/helpers/class-param.php:28
Classic_SEO\H\Param::get()
wp-content/plugins/classicpress-seo/includes/admin/helpers/class-page.php:268
Classic_SEO\A\Page::is_current_page()
wp-content/plugins/classicpress-seo/includes/module/class-base.php:36
Classic_SEO\M\Base::__construct()
wp-content/plugins/classicpress-seo/includes/modules/status/class-status.php:48
Classic_SEO\S\Status::__construct()
wp-content/plugins/classicpress-seo/includes/module/class-manager.php:314
Classic_SEO\M\Manager::load_module()
wp-content/plugins/classicpress-seo/includes/module/class-manager.php:294
Classic_SEO\M\Manager::load_modules()
wp-includes/class-wp-hook.php:286
WP_Hook::apply_filters()
wp-includes/class-wp-hook.php:310
WP_Hook::do_action()
wp-includes/plugin.php:453
do_action()
wp-settings.php:330
require_once()
wp-config.php:158
require_once()
wp-load.php:37
require_once()
wp-admin/admin.php:31
```

### PR DESCRIPTION
**Pull request by xxsimoxx**
First step to PHP 8 compatibility.

- Changed the lines throwing errors
- Rebuilt using `composer install`
- Updated Update Manager client
- Bumbed version

Fixes #137

### CODE REVIEW

File: classicpress-seo.php [hunk 1]
```diff
@@ -142,14 +143,14 @@ public function __call( $name, $arguments ) {
...
+
 	/**
 	 * Initialize.
 	 */
 	public function init() {
-		
+
 	}
-	
+
```
Comment (bahiirwa) [inline classicpress-seo.php:153]: Should we consider removing this method as we do not need it?

**Comment by bahiirwa** [comment]
This looks in order apart from the empty method `init()` we keep carrying over. Do we utilize it anywhere?

File: classicpress-seo.php (hunk 1 above)
Comment (xxsimoxx) [reply classicpress-seo.php:153]: Don't know why it is there, so I've left it even if does nothing.

**Review by bahiirwa — APPROVED** [review]
LGTM

**Comment by viktorix** [comment]
Were you able to run it through CPCS to see if any issues come up? Also, PHPCS with version compatibility?

**Comment by xxsimoxx** [comment]
No, CPCS is not passing. 
PHPCompatibility looks good with 5.6, 7.0 and 7.4.

**Comment by viktorix** [comment]
> No, CPCS is not passing.

Is it enough to pass the directory review process?

> PHPCompatibility looks good with 5.6, 7.0 and 7.4.

Are we able to check with 8.0? Just to be sure.

**Comment by xxsimoxx** [comment]
[PHPCompatibility](https://github.com/PHPCompatibility/PHPCompatibility) seems stalled so PHP 8 checks are already under development.

**Comment by xxsimoxx** [comment]
As discussed now UPDATE_SERVER points again to https://www.cpseo.net.

**Review by viktorix — APPROVED** [review]
Everything looks good. Go ahead and merge.

**Pull request merged**